sktime/utils/_testing/series.py

```diff
@@ -62,3 +62,33 @@ def _make_index(n_timepoints, index_type=None):
 
     else:
         raise ValueError(f"index_class: {index_type} is not supported")
+
+
+def _load_solar(
```

Comment:
Nice!

I would actually give this a docstring, and move it to `datasets` as a "full" data set, not just one for testing.

* question: is this reliable, as in, always producing the same data?
* I would suggest to add a test for this
* reference should be added in `docs/source/api_reference/datasets`

Reply:
Good shout, update now including a test